lib/common/util/json.js

```diff
@@ -11,3 +13,17 @@ export function isValidJSON (str: string): boolean {
   }
   return true
 }
+
+/**
+ * Check if a string is valid JSONC.
+ */
+export function isValidJSONC (str: string): boolean {
```

Comment:
Sorry for not catching this earlier. But what is the JSONC parser supposed to permit?  Can you provide a little more detail either here or in the CustomConfig component?

![image](https://user-images.githubusercontent.com/2370911/75889204-8dc61e80-5dfa-11ea-9fe6-be300acd2c5d.png)

Reply:
It might be good to ref the function in OTP that parses JSON and match what is allowed there: https://github.com/opentripplanner/OpenTripPlanner/blob/27f4ed0a86157bdd4c4bc3004fec25687768d373/src/main/java/org/opentripplanner/standalone/OTPMain.java#L190-L194